CMSgov/cms-carts-seds
# MDCT-2925: FFY 2023 Content Changes (Section 4)

### Description

Content updates to reflect copy for FFY 2023 in Section 4 of CARTS.

### Related ticket(s)


MDCT-2925

---
### How to test

- Log in as `cms.admin@test.com`
- Generate form base templates for both 2022 and 2023
- Log in as `stateuser2@test.com`
- Navigate to the 2023 form, Section 4

Verify that the content in red (on the ticket) matches the content in the form

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
